transports/sql/transactions_ambient-core-warning_sqltransport_[4,).partial.md

```diff
@@ -0,0 +1 @@
+WARNING: The Transaction scope mode is not available in .NET Core 2.0 because the implementation of `SqlConnection` does not support enlisting in an ambient transaction.
```

Comment:
`SqlConnection` throws at runtime. This is because MS claims it will probably work (to some degree) in 2.1.